x-pack/plugins/infra/public/components/logging/log_time_controls.tsx

```diff
@@ -57,7 +61,10 @@ export class LogTimeControls extends React.PureComponent<LogTimeControlsProps> {
             />
           </InlineWrapper>
           <EuiFilterButton iconType="play" iconSide="left" onClick={this.startLiveStreaming}>
-            Stream live
+            <FormattedMessage
+              id="xpack.infra.logs.streamLiveButtonLabel"
```

Comment:
How about `xpack.infra.logs.startStreamingButtonLabel` to maintain symmetry to the stop button label?

Reply:
Change it